client/src/templates/Introduction/components/cert-challenge.tsx

```diff
@@ -47,12 +45,16 @@ const honestyInfoMessage = {
 };
 
 const mapStateToProps = (state: unknown) => {
+  const currentCerts = certificatesByNameSelector(
+    // @ts-expect-error One day, state will be typed
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    state?.app?.appUsername || ''
+  )(state)?.currentCerts;
```

Comment:
This is a bit puzzling.  I'd expect to see a function of the form
```
(state) => stuff
```
but this is just `stuff`.

Reply:
It was odd, but do you see something particularly incorrect?

Reply:
Sorry, I missed this.  No, I don't necessarily think it's incorrect, but it's very different to every other selector used here.  Unless it needs to be different, it shouldn't be written differently.

Reason being, it forces people to either ignore the weirdness or try and figure out *why* it's weird.  Neither is good.

Reply:
TL;DR: it should be defined inside `redux` like all the other ones.